<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="pelneImieLabel">
       <property name="text">
        <string>Pełne imię</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="full_name_edit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="dataUrodzeniaLabel">
       <property name="text">
        <string>Data urodzenia</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="dataZgonuLabel">
       <property name="text">
        <string>Data zgonu</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="krajLabel">
       <property name="text">
        <string>Kraj</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="country_edit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="imieWMPCLabel">
       <property name="text">
        <string>Imię w MPC</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="name_mpc_edit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="birth_date_edit"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="death_date_edit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="error_label">
     <property name="styleSheet">
      <string notr="true">color: rgb(239, 41, 41);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
